creatorkit/Project 1.md

```diff
@@ -0,0 +1,40 @@
+![Creator Logo](../creatorlogo.png)
+
+# Project 1 - IoT Button
+
+## Introduction
+
+This project will allow you to press a button on one of your Clicker boards and see an LED toggle on the other Clicker board. Ci40 acts as the hub for the 6LoWPAN communication. You can also view the inter-device messages via the FlowM2M website - demonstrating connectivity with the cloud.
```

Comment:
Should this be "demonstrating device registration and connectivity with the cloud" ?